apps/react-app/src/pods/manfred-json-input/manfred-json-input.container.tsx

```diff
@@ -1,6 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
 import { ManfredJsonInput } from './manfred-json-input.component';
+import { switchRoutes } from '@/core';
 
 export const ManfredJsonInputContainer: React.FC = () => {
-  return <ManfredJsonInput />;
+
+  const [data, setData] = useState('');
```

Comment:
Remove this useState, is not used, is just dead code.

Why is this not used? Because we are handling all the temp json data storage in the component, not the container, the contain is just receiving the final piece of data and navigating to export (later on we will store this in a context, next case ;))